pulsar-broker/src/main/java/org/apache/pulsar/broker/ManagedLedgerClientFactory.java

```diff
@@ -102,7 +102,15 @@ public void initialize(ServiceConfiguration conf, ZooKeeper zkClient,
         };
 
         this.managedLedgerFactory =
-                new ManagedLedgerFactoryImpl(bkFactory, zkClient, managedLedgerFactoryConfig, statsLogger);
+                createManagedLedgerFactory(zkClient, managedLedgerFactoryConfig, statsLogger, bkFactory);
+    }
+
+    protected ManagedLedgerFactoryImpl
+    createManagedLedgerFactory(ZooKeeper zkClient,
```

Comment:
nit: fix indent

Reply:
My IDE deal with the indent for me, I'm not sure what's the meaning of your 'fix indent' here. But I think the reason why the indent looks a little weird is that `BookkeeperFactoryForCustomEnsemblePlacementPolicy` is too long, the IDE has to format like this to make a line less than 120 chars.